cluster-autoscaler/processors/status/scale_down_status_processor.go

```diff
@@ -33,6 +35,7 @@ type ScaleDownStatus struct {
 	UnremovableNodes  []*UnremovableNode
 	RemovedNodeGroups []cloudprovider.NodeGroup
 	NodeDeleteResults map[string]NodeDeleteResult
+	ObtainedAt        time.Time
```

Comment:
The name suggests that this would be the timestamp that the status was obtained at, but the implementation puts the timestamp of the latest node delete result instead. This was quite confusing while reading. Could we change it to something more accurate, like `NodeDeleteResultsLatestTimestamp`, or `LatestNodeDeleteResultObtainedAt`?